lib/routes/sign.js

```diff
@@ -101,10 +103,25 @@ module.exports = function (log, isA, error, signer, db, domain) {
         ).then(function(certResult) {
           log.activityEvent('account.signed', request, {
             uid: uid,
+            network: getNetworkHash(uid, request.info.remoteAddress),
+            device: getDeviceHash(sessionToken.id),
             account_created_at: sessionToken.accountCreatedAt
           })
```

Comment:
(also then it's not sitting in the same event as the `uid` that was used to salt the hash)